api/src/backend/api/specs/v1.yaml

```diff
@@ -417,6 +470,7 @@ paths:
         name: filter[delta]
         schema:
           type: string
+          x-spec-enum-id: fef6b0e2506af8bc
```

Comment:
This is the only thing that concerns me. This enum ID probably changes every time we build the API container, so we will see differences in the views each time we download the spec. We should take this into account and avoid uploading this kind of change